client/js/table-model.js

```diff
@@ -16,6 +16,21 @@ class TableModel {
   setValue(location, value) {
     this.data[this._getCellId(location)] = value;
   }
+
+  getSum(column) {
+    let sum = [];
+    for (var prop in this.data) {
```

Comment:
This approach to fetching the data is very inefficient, because it potentially visits every cell in the sheet just to get one column of data.  Another reason to not use the `for..in` style to directly manipulate `this.data`.